doc/modules/groups.h

```diff
@@ -38,6 +38,14 @@
  */
 
 /**
+ * @defgroup InitialTopographyModels Describing the properties of the initial topography of the domain
+ *
+ * A module for the definition of properties initial topography of the geometry.
```

Comment:
grammar. Maybe "A module for the definition of properties RELATED TO the topography of the geometry."?